.github/workflows/check-unity-sdk-for-updates.yml

```
@@ -0,0 +1,52 @@
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v7
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: update-adgem-sdk
+          title: 'build: update adgem sdk dependency version'
+          body: 'This PR updates the AdGem SDK version to the latest release.'
+          labels: 'automated pr'
+          assignees: |
+            tustanivsky
+            munkiki7
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)